src/Popper.test.js

```diff
@@ -71,7 +71,7 @@ describe('Popper component', () => {
     const referenceElement = document.createElement('div');
     expect(() =>
       mount(
-        <InnerPopper referenceElemen={referenceElement}>
+        <InnerPopper referenceElement={referenceElement}>
```

Comment:
Thanks to `typo` test was passing. Not a `flow` user... but typescript would throw an error `Property 'referenceElemen' does not exist on type ...`

Reply:
referenceElem is not disallowed by the component. If TS thrown that kind of error it's wrong

Reply:
We have `referenceElement` on PopperProps not `referenceElemen` as it looks like typo, just was wondering way flow don't warn about it... but not related to this PR

Reply:
Yes I noticed it. It doesn't warn because the typoed prop is not disallowed explicitly